motivator/users/views.py

```diff
@@ -0,0 +1,64 @@
+
+import re
+from django.shortcuts import redirect, render
+from django.views.generic import CreateView
+from .forms import UserCreationForm, MotivationForm
+from django.contrib.auth import authenticate, login
+from .models import Motivation, User
+from django.conf import settings
+
+
+class Register(CreateView):
+    template_name = 'registration/register.html'
+
+    def get(self, request):
+        context = {
+             'form': UserCreationForm() 
+            }
+        return render(request, self.template_name, context)
+
+    def post(self, request):
+        form = UserCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get('username')
+            password = form.cleaned_data.get('password1')
+            user = authenticate(username=username, password=password)
+            login(request, user)
+            return redirect('home')
+        context = {
+            'form': form
+        }
+        return render(request, self.template_name, context)
+
+def list(request):
+    motivations = Motivation.objects.all()
```

Comment:
Not good here. In a case when you have a lot of items in `Motivation` table you'll experience performance issues. Way to optimize is to paginate